.github/workflows/mypy.yaml

```diff
@@ -0,0 +1,42 @@
+name: Run mypy
```

Comment:
There is a mypy reusable workflow but I guess you can't use it because the version file created will not be the right one. Maybe you can create the version file and then call the reusable workflow? Else @mwittgen might need to add a parameter to the workflow with the version path.

Reply:
Indeed, `version.py` gives me troubles. I checked workflow and could not figure a way to pre-create one at the right location. If there is a way to configure the location, it would work for me.